app/auth/views/oidc.py

```diff
@@ -16,14 +18,15 @@
 from app.email_utils import send_welcome_email
 from app.log import LOG
 from app.models import User, SocialAuth
-from app.utils import encode_url, sanitize_email, sanitize_next_url
+from app.utils import sanitize_email, sanitize_next_url
 
 
 # need to set explicitly redirect_uri instead of leaving the lib to pre-fill redirect_uri
 # when served behind nginx, the redirect_uri is localhost... and not the real url
 _redirect_uri = URL + "/auth/oidc/callback"
```

Comment:
Nitpicking. Can you rename this to `redirect_uri`? We don't need to keep the `_` prefix since it's already the url we're going to use.